src/data/object_events/movement_action_func_tables.h

```diff
@@ -266,6 +266,12 @@ u8 MovementAction_FlyUp_Step1(struct ObjectEvent *, struct Sprite *);
 u8 MovementAction_Fly_Finish(struct ObjectEvent *, struct Sprite *);
 u8 MovementAction_FlyDown_Step0(struct ObjectEvent *, struct Sprite *);
 u8 MovementAction_FlyDown_Step1(struct ObjectEvent *, struct Sprite *);
+//slow running
```

Comment:
Nitpicking, but this file's comments seem to arbitrarily alternate between "slow running" and "run slow". I'd personally pick one and generalise it